monitoring/uss_qualifier/requirements/astm/f3548/v21.md

```diff
@@ -0,0 +1,167 @@
+# ASTM F3548-21
+
+_Standard Specification for UAS Traffic Management (UTM) UAS Service Supplier (USS) Interoperability_
+
+For information on these requirements, refer to [the ASTM standard F3548-21](https://www.astm.org/f3548-21.html).
+
+## Common Requirements
+
+* <tt>GEN0005</tt>
+* <tt>GEN0010</tt>
+* <tt>GEN0015</tt>
+* <tt>GEN0100</tt>
+* <tt>GEN0105</tt>
+* <tt>GEN0200</tt>
+* <tt>GEN0300</tt>
+* <tt>GEN0305</tt>
+* <tt>GEN0310</tt>
+* <tt>GEN0400</tt>
+* <tt>GEN0405</tt>
+* <tt>GEN0500</tt>
```

Comment:
I know the checklist lists GEN0500 in those two services, but it is defined in section 5.2 (specifically 5.2.7.1) which was the rationale behind putting it here.  I think the checklists in the standard may not be fully accurate, so I've tried to make a few changes in this definition document so that each requirement is only specified once (this would be important if we were documenting what the requirement was, even though we wouldn't do this with the standards since the standard is the source of truth for that).  Now most requirements should appear in the section of the standard in which they are defined.  A few (USS*) are lumped into other sections rather than breaking them out into another "USS-USS Interfaces" section where they are actually defined in the standard because I think that's more helpful, but that could be changed in the future if it causes problems.